packages/core/admin/server/.eslintrc

```diff
@@ -2,6 +2,6 @@
   "root": true,
   "extends": ["custom/back/typescript"],
   "parserOptions": {
-    "project": ["./server/tsconfig.eslint.json"]
```

Comment:
isn't this necessary as the config is based on the root folder of the package ? Can you check what we do elsewhere ?

Reply:
Only 4 of them were doing this like that and it was breaking the auto parsing in the IDE, so I pushed that to see if it still works in the CI. I'm following this closely :slightly_smiling_face:

Reply:
Yep, it seems it's failing in the CI now. I'll have to go without eslint locally, I guess :upside_down_face: 

I'll revert the changes